packages/ingest/package.json

```diff
@@ -50,15 +50,16 @@
     "cksum": "^1.3.0",
     "encodeurl": "^1.0.2",
     "fs-extra": "^5.0.0",
-    "got": "^11.8.5",
+    "got": "^14.2.0",
```

Comment:
Got > 12 was required for node v18+, and there were several bugfixes that made got v14 required.